src/components/PasswordForget/PasswordForget.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+
+import {withFirebase} from '../Firebase';
+import * as ROUTES from '../../routes';
+
+const PasswordForgetPage = () => (
+  <div>
+    <h1>Password forget</h1>
+    <PasswordForgetForm />
+  </div>
+);
+
+const INITIAL_STATE = {
+  email: '',
+  error: null
+};
+
+class PasswordForgetFormBase extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {...INITIAL_STATE};
```

Comment:
I think `this.state = { ...INITIAL_STATE }` is the same thing as` this.state = INITIAL_STATE`

Reply:
I am not too sure, but will have a look.